scipy/stats/mstats_basic.py

```diff
@@ -549,6 +549,42 @@ def _spearmanr_2cols(x):
         return SpearmanrResult(rs, prob)
 
 
+def _kendall_p_exact(n, c):
+    # Exact p-value, see Maurice G. Kendall, "Rank Correlation Methods" (4th Edition), Charles Griffin & Co., 1970.
+    if n <= 0:
+        raise ValueError
```

Comment:
This line may have been moved from elsewhere but probably should be something like:
  ValueError(f'n ({n}) must be positive')

Reply:
Thanks. Fixed!